networking_oneview/ml2/drivers/oneview/common.py

```diff
@@ -799,6 +803,7 @@ def check_unique_lig_per_provider_constraint(uplinkset_mappings):
 
 
 def delete_outdated_flat_mapped_networks(flat_net_mappings):
+    LOG.info("Synchronizing flat network mappings")
```

Comment:
Why not Synchronize, to keep the same pattern as the other Logs.

Reply:
+1